docs/products/opensearch/concepts/opensearch_backups.rst

```diff
@@ -0,0 +1,24 @@
+OpenSearch backups
+==================
+
+Aiven for OpenSearch databases are automatically backed up, `encrypted <https://help.aiven.io/en/articles/977466-cloud-security-overview>`_, and stored securely in object storage.
+
+Depending on the service plan, we offer single backups for disaster recovery or daily backups with different retention periods:
```

Comment:
This could be worth including in a generic topic, and linking from here.

Reply:
It could be, although I think there might be some variation across different products...